locales/en/messages.po

```diff
@@ -2775,7 +2775,7 @@ msgid "justfix.commonWarningAboutChangingLandlordDetails"
 msgstr "If you receive rent-related documents from a different address, or if your landlord has instructed you to use a different address for correspondence, you can <0>provide your own details.</0> But, only use a different address if you are absolutely sure it is correct— it is safer to use the official address your landlord provided to the city."
 
 #: frontend/lib/data-driven-onboarding/data-driven-onboarding.tsx:337
-msgid "justfix.ddoEfnycCovidMessage"
+msgid "justfix.ddoEvictionfreeCovidMessage"
```

Comment:
Should this be `Evictionfree` or `EvictionFree`?

Reply:
Oh good question, I think `Evictionfree` is fine.

We are a bit inconsistent here with NoRent, sometimes calling it `Norent` in variable names and other times calling it `NoRent`.  This is partly because my policy with converting snake-case to camel-case is by replacing all underscores with capital letters, but in snake case we call these things `EVICTIONFREE` and `NORENT` so I felt there wasn't a need to capitalize anything but the first letter... ah well.